src/Objects/_PVector.js

```diff
@@ -0,0 +1,283 @@
+module.exports = function(options, undef) {
+  'use strict';
+
+  const TAU = 2 * Math.PI,
+        //TAU = options.PConstants.TAU,
```

Comment:
It does, it's a less efficient way of passing objects around. When the code was ported to CommonJS modules, the initial rewrite couldn't quite rely on `require()` calls, so a lot of things ended up being put in `options` and passed around that way. However, any new refactors/files should be able to use the proper CommonJS require system instead. There's a lot of places where `options` can be removed and direct requires can now be used, so a fair number of issues need to be filed (by me, or anyone willing to file them) to identify all the places where `options` should be replaced by proper requiring.